gnomad_qc/v4/annotations/insilico_predictors.py

```diff
@@ -0,0 +1,117 @@
+        - gnomad 3.0 indels: `cadd.v1.6.indels.tsv.bgz` (1.1G) downloaded from `https://krishna.gs.washington.edu/download/CADD/v1.6/GRCh38/gnomad.genomes.r3.0.indel.tsv.gz`. It contains 100,546,109 indels in gnomaD v3.0.
+        - gnomad 3.1 indels: `CADD-indels-gnomad.3.1.ht` was run by William Phu with CADD v1.6 in 2020. It contains 166,122,720 indels in gnomAD v3.1.
+        - gnomad 3.1 complex indels: `CADD-1.6-gnomad-complex-variants.ht` was run by William Phu with CADD v1.6 in 2020. It contains 2,307 complex variants that were not filtered out by Hail is.indels().
+        - gnomAD v4 indels: `cadd.v1.6.gnomAD_v4_new_indels.tsv.bgz` (368M) was run by Qin He with CADD v1.6 in 2023. It contains 32,561,253 indel that are new in gnomAD v4.
+        - 1,972,208 indels were duplicated in gnomAD v3.0 and v4.0, in gnomAD v3.1 and v4.0. However, CADD only generate a score per loci. We use anti_join to keep only the latest prediction.
+    :return: Hail Table with CADD scores for GRCh38.
+    """
+
+    def load_cadd_raw(cadd_tsv) -> hl.Table:
+        column_names = {
+            "f0": "chr",
+            "f1": "pos",
+            "f2": "ref",
+            "f3": "alt",
+            "f4": "RawScore",
+            "f5": "PHRED",
+        }
+        types = {"f0": hl.tstr, "f1": hl.tint32, "f4": hl.tfloat32, "f5": hl.tfloat32}
+        ht = hl.import_table(
+            cadd_tsv,
+            delimiter="\t",
+            types=types,
+            no_header=True,
+            force_bgz=True,
+            comment="#",
+            min_partitions=1000,
+            missing="NA",
+        )
+        ht = ht.rename(column_names)
+        chr = hl.if_else(ht.chr.startswith("chr"), ht.chr, hl.format("chr%s", ht.chr))
+        ht = ht.annotate(
+            locus=hl.locus(chr, ht.pos, reference_genome="GRCh38"),
+            alleles=hl.array([ht.ref, ht.alt]),
+        )
+        ht = ht.select("locus", "alleles", "RawScore", "PHRED")
+        ht = ht.key_by("locus", "alleles")
+
+        return ht
+
+    snvs = load_cadd_raw(
+        "gs://gnomad-qin/v4_annotations/cadd.v1.6.whole_genome_SNVs.tsv.bgz"
+    )
+    indel3_0 = load_cadd_raw(
+        "gs://gnomad-qin/v4_annotations/cadd.v1.6.gnomad.genomes.r3.0.indel.tsv.bgz"
+    )
+    indel3_1 = hl.read_table("gs://gnomad-wphu/CADD-indels-gnomad.3.1.ht")
+    indel3_1_complex = hl.read_table(
+        "gs://gnomad-wphu/CADD-1.6-gnomad-complex-variants.ht"
+    )
+    indel4 = load_cadd_raw(
+        "gs://gnomad-qin/v4_annotations/cadd.v1.6.gnomAD_v4_new_indels.tsv.bgz"
+    )
+
+    # Merge the CADD predictions run for v3 versions.
+    indel3 = indel3_0.union(indel3_1, indel3_1_complex)
+
+    # This will avoid duplicated indels in gnomAD v3 and v4.
+    indel3 = indel3.anti_join(indel4)
+
+    ht = snvs.union(indel3, indel4)
+    return ht
```

Comment:
Can you add a cadd version to as a global here which would eventually be put on the final in silico HT? This is something we'll want on the final release HT for all in silico versions.